ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -11,9 +11,8 @@ pipeline {
         SKUBA_BINPATH = '/home/jenkins/go/bin/skuba'
         VMWARE_ENV_FILE = credentials('vmware-env')
         GITHUB_TOKEN = credentials('github-token')
-        PLATFORM = 'vmware'
         TERRAFORM_STACK_NAME = "${BUILD_NUMBER}-${JOB_NAME.replaceAll("/","-")}".take(70)
-        PR_CONTEXT = 'jenkins/skuba-test'
+        PR_CONTEXT = 'jenkins/skuba-test-${PLATFORM}'
```

Comment:
Note this is likely to break PRs requirements in skuba github repo as there will not be anymore a `skuba-test`, which is a required one at the moment, not sure how and where is this configured, I guess on github UI.

Reply:
github-ui -> protected branches I think :)

Reply:
This is no longer the case, I changed the PR to be really non intrusive just creating a new job with new PR context and do not touch anything else.